lib/voronoi/Install.py

```diff
@@ -18,7 +18,7 @@
 # settings
 
 version = "voro++-0.4.6"
-url = "http://math.lbl.gov/voro++/download/dir/%s.tar.gz" % version
+url = "https://download.lammps.org/thirdparty/%s.tar.gz" % version
```

Comment:
I think, something like the Debian or Gentoo mirror might be better here, e.g. http://distfiles.gentoo.org/distfiles/voro++-0.4.6.tar.gz!